opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
ok I get it now. basically this would be documented as a buddy of joinTrace, which would be ancillary to routine tasks like addTag. Implementations would implement this to make their choices wrt which (if any) attributes to pass down. Is the returned childContext immutable in this design? Or would we expect someone to do something like... 

```java
childContext = currentSpan.childContext();
childContext.setAttribute("foo", "bar");
Tracer.joinTrace("new-span-name", childContext);
```

Or are we just saying that this is the hook to make babies.